app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def showLanding  
+    render template: "landing" 
```

Comment:
It would be slightly more standard to rename "landing" to "index"

Reply:
This would be also changing `app/views/landing.html.erb` to `app/views/index.html.erb`